modules/openapi-generator/src/main/resources/dart2/api_client.mustache

```diff
@@ -147,9 +146,48 @@ class ApiClient {
 
     throw ApiException(HttpStatus.badRequest, 'Invalid HTTP operation: $method $path',);
   }
+{{#native_serialization}}
+
+  Future<dynamic> deserializeAsync(String json, String targetType, {bool growable}) async =>
+    // ignore: deprecated_member_use_from_same_package
+    deserialize(json, targetType, growable: growable);
+
+  @Deprecated('Scheduled for removal in 6.0. Use deserializeAsync() instead.')
+  dynamic deserialize(String json, String targetType, {bool growable}) {
+    // Remove all spaces. Necessary for regular expressions as well.
+    targetType = targetType.replaceAll(' ', ''); // ignore: parameter_assignments
+
+    // If the expected target type is String, nothing to do...
+    return targetType == 'String'
+      ? json
+      : _deserialize(jsonDecode(json), targetType, growable: growable == true);
+  }
+{{/native_serialization}}
+
+  // ignore: deprecated_member_use_from_same_package
+  Future<String> serializeAsync(Object value) async => serialize(value);
+
+  @Deprecated('Scheduled for removal in 6.0. Use serializeAsync() instead.')
```

Comment:
minor suggestion: replace `6.0` with `OpenAPI Generator 6.x`

as users may have no idea what `6.0` refers to

Reply:
Most definitely, coming up.